language-plutus-core/src/CostModel.md

```diff
@@ -43,3 +43,17 @@ How to make a new model:
 - `ExBudgeting.hs`: add the haskell version of the predictor
 - `CostModelCreation.hs`: add code to load the model from R code
 - Fix compile errors until `language-plutus-core-test-cost-model` runs & run it.
+
+## Calibration
+
+## CPU
+
+The CPU calibration is according to the time used by a builtin operation as determined by criterion. The cost per evaluation step isn't determined yet.
+
+## Memory
+
+The memory cost per builtin operation on integers is the expected worst-case cost of the generated value. This way the computation stays simple. If Plutus were to adjust the usage down to actual cost, the max and the current usage would have to be carried around separately.
```

Comment:
Worth saying that we don't account for memory allocated inside foreign functions, and why.

Reply:
That goes into the extended documentation. We do account for it, it just happened to be 0 in almost all cases here.